effekt/shared/src/main/scala/effekt/core/PolymorphismBoxing.scala

```diff
@@ -198,7 +198,7 @@ object PolymorphismBoxing extends Phase[CoreTransformed, CoreTransformed] {
       if (coerce.isIdentity) {
         List(Definition.Let(id, transform(tpe), transform(binding)))
       } else {
-        val orig = TmpValue()
+        val orig = TmpValue("letDefinition")
```

Comment:
This is the name of a binding to be boxed/unboxed. So a long name would be `toBeCoerced`. Alternatively, also a short name like `x` would work. - it should be very clear from the (immediately) following call to the coercer that this is what's happening.